### ISSUE TITLE
Livid DS1 problem assignments in Max for Live 8.5

### ISSUE THREAD

**Issue by Sheld0n-Cooper** [aumhaa/m4m8#11]
Hello, I would like to use the livid ds1 in max for live 8.5. I cannot map the buttons of the livid DS1 and I have an error:
"midi_mme: error 7 opening midi input device DS1"
"midi_mme: error 7 opening output device DS1"

The problem can be solved if I use your Livid DS1 v2 script.
But if I do that the Livid DS1 buttons are already mapped to features, and that's not what I want since I want to make my own assignments in Max for Live. What can I do ?
Can you give me a script where there are no assignments please?

Edit:

I did another test without putting your Livid Ds1 script.
I launched Max fo live then I tried to make a noon assignment, it didn't work. I then disconnected the livid DS1 then I reconnected it, the midi affections works after the reconnection.

So, in summary, the Livid Ds1 works in Max for Live, but you have to unplug it and plug it back in. This is quite abnormal behavior.
Can you help me solve the problem?

Thank you in advance.
